alerta/auth/basic_ldap.py

```diff
@@ -68,82 +64,83 @@ def login():
     #       Except: Search query is bad defined
     # Else
     #   Set the DN as the one found in LDAP_DOMAINS variable
-    domain_search_query = current_app.config.get('LDAP_DOMAINS_SEARCH_QUERY', {})
-    base_dns = current_app.config.get('LDAP_DOMAINS_BASEDN', {})
-    user_base_dn = current_app.config.get('LDAP_DOMAINS_USER_BASEDN', {})
-    if domain in domain_search_query:
-        ldap_bind_username = current_app.config.get('LDAP_BIND_USERNAME', '')
-        ldap_bind_password = current_app.config.get('LDAP_BIND_PASSWORD', '')
+    user_filter = current_app.config['LDAP_USER_FILTER']
+    user_base_dn = current_app.config['LDAP_USER_BASEDN']
+    if user_filter:
+        ldap_bind_username = current_app.config['LDAP_BIND_USERNAME']
```

Comment:
I'm assuming current_app.config doesn't fail if that entry isn't in the array ?

Reply:
if it does - it should probably be moved into the try :)

Reply:
As far as I can see this cannot fail because it is already part of ```alerta/settings.py``` file, so the property will always exist although it will be blank string if it is not added in the user config file

Reply:
@frasy is right. As long a default value already exists in `settings.py` it won't throw a key error.